src/main/java/uk/gov/justice/hmpps/prison/api/model/BedAssignment.java

```diff
@@ -52,4 +52,7 @@
 
     @ApiModelProperty(value = "Bed assignment sequence. Used as a primary key when combined with the booking id", position = 10, example = "2")
     private Integer bedAssignmentHistorySequence;
+
+    @ApiModelProperty(value = "the staff member responsible for the movement of a prisoner", position = 11, example = "KQJ74F")
+    private String createUserId;
```

Comment:
Is this the `staffId` or `username`?  If the latter can it be called `createUsername` instead please?  If the former then the `example` would need to be updated

Reply:
it is an Id value and not a user name value, I got the value 'KQJ74F' from the data table, please suggest a better example and I'll be happy to include it

Reply:
shall I use 'SA' as an example?